Makefile

```diff
@@ -3,7 +3,7 @@
 TOPDIR = $(CURDIR)
 
 # Directories which require calling make recursively
-SUBDIR = sys lib bin usr.bin etc include
+SUBDIR = sys lib bin usr.bin etc include scripts
```

Comment:
I don't think we need to introduce new subdirectory just for scripts. Please move our `ps` to `bin/ps` where it belongs to.

Reply:
done